packages/runtime_events_tools/runtime_events_tools.0.5.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Tools for the runtime events tracing system in OCaml"
+description: "Various tools for the runtime events tracing system in OCaml"
+maintainer: ["Sadiq Jaffer" "KC Sivaramakrishnan" "Sudha Parimala"]
+authors: ["Sadiq Jaffer"]
+license: "ISC"
+homepage: "https://github.com/tarides/runtime_events_tools"
+bug-reports: "https://github.com/tarides/runtime_events_tools/issues"
+depends: [
+  "dune" {>= "3.2"}
+  "ocaml" {>= "5.0.0~"}
+  "ocamlfind"
+  "hdr_histogram"
+  "cmdliner"
+  "tracing"
+  "menhir" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/tarides/runtime_events_tools.git"
+url {
+  src:
+    "https://github.com/tarides/runtime_events_tools/releases/download/0.5.0/runtime_events_tools-0.5.0.tbz"
+  checksum: [
+    "sha256=818258e2f81d42327cee846dda9fec5a863e926152903be3f4e5c686ee660270"
+    "sha512=50c5ecdae4a73c4f26d68da21265c8ee2731bff76445db9e637237793d7024a272f78f6a57650e51c70a459843a656c19d5ee79d2e5aeed0460983adb67fe548"
+  ]
+}
```

Comment:
```suggestion
}
available: os != "freebsd"
```

CI suggests that `hdr_histogram` is not available on freebsd

```
# /usr/home/opam/.opam/5.1.0/.opam-switch/build/hdr_histogram.0.0.3/_build/default/lib/libhdr_histogram/src/hdr_time.c:80:2: warning: #warning is a language extension [-Wpedantic]
# #warning "Platform not supported\n"
#  ^
# /usr/home/opam/.opam/5.1.0/.opam-switch/build/hdr_histogram.0.0.3/_build/default/lib/libhdr_histogram/src/hdr_time.c:80:2: warning: "Platform not supported\n" [-W#warnings]
# 2 warnings generated.
```

Reply:
Yes. Should I explicitly disable freebsd in this package too?

Reply:
yes I think we should mark it as unavailable for freebsd https://github.com/ocaml/opam-repository/pull/24530/files#r1345486302